nidx/nidx_text/src/schema.rs

```diff
@@ -46,6 +46,7 @@ pub struct TextSchema {
     pub groups_with_access: Field,
 
     pub encoded_field_id: Option<Field>,
+    pub encoded_field_id_bytes: Option<Field>,
```

Comment:
We will want to make `encoded_field_id` (the previous one), not an `Option` at some point. Can be this PR or later.